### PR TITLE
Toml/doc db-backends conf update

### PR DESCRIPTION
**Pull request by gustawlippa**
This PR updates the `doc/advanced-configuration/database-backends-configuration.md `.
I also think there was an error in the listeners docs concerning the path.
I'm not sure if the provided configuration for MS SQL is correct

### CODE REVIEW

File: doc/advanced-configuration/database-backends-configuration.md [hunk 1]
```diff
@@ -206,15 +206,16 @@ cat azuresql.sql | tr -d '\r' | tr '\n' ' ' | sed 's/GO/\n/g' |
...
+Configure the `outgoing_pools.rdbms` section as follows:
 
-```erlang
-{rdbms_server_type, mssql}.
-{outgoing_pools, [
- {rdbms, global, default, [{workers, 5}],
-  [{server, "DSN=mongoose-mssql;UID=username;PWD=password"}]}
-]}.
+```toml
+[outgoing_pools.rdbms.default]
```
Comment (arcusfelis) [inline doc/advanced-configuration/database-backends-configuration.md:213]: `rdbms_server_type = "mssql"` is still required I think.

Comment (chrzaszcz) [reply doc/advanced-configuration/database-backends-configuration.md:213]: Yes, put it in the `general` section

File: doc/advanced-configuration/database-backends-configuration.md [hunk 2]
```diff
@@ -337,7 +339,7 @@ curl -X PUT $ELASTICSEARCH_URL/muc_messages -d '@priv/elasticsearch/muc.json'
 
 where `$ELASTICSEARCH_URL` is a URL pointing to your ElasticSearch node's HTTP API endpoint.
 
-Please refer to [advanced configuration](../Advanced-configuration.md#elasticsearch-connection-setup) page to check how to configure MongooseIM to connect to ElasticSearch node.
+Please refer to [advanced configuration](../advanced-configuration/outgoing-connections.md#elasticsearch-options) page to check how to configure MongooseIM to connect to ElasticSearch node.
```
Comment (chrzaszcz) [inline doc/advanced-configuration/database-backends-configuration.md:342]: _the_ advanced configuration page

File: doc/advanced-configuration/database-backends-configuration.md [hunk 3]
```diff
@@ -83,8 +83,8 @@ mysql -h localhost -u user -p -e 'create database mongooseim'
 mysql -h localhost -u user -p mongooseim < mysql.sql
 ```
 
-You should also configure MySQL database in the `mongooseim.cfg` file.
-Please refer to the [Advanced configuration/Database setup](../Advanced-configuration.md) for more information.
+You should also configure MySQL database in the `mongooseim.toml` file.
```
Comment (chrzaszcz) [inline doc/advanced-configuration/database-backends-configuration.md:86]: _the_ MySQL database

File: doc/advanced-configuration/database-backends-configuration.md [hunk 4]
```diff
@@ -120,8 +120,8 @@ You can use the following command to apply it on localhost:
 psql -h localhost -U user -c "CREATE DATABASE mongooseim;"
 psql -h localhost -U user -q -d mongooseim -f pg.sql
 ```
-You should also configure Postgres database in `mongooseim.cfg` file.
```
Comment (chrzaszcz) [inline doc/advanced-configuration/database-backends-configuration.md:123]: _the_ Postgres database in _the_ `mongooseim.cfg` file.

**Review by chrzaszcz — COMMENTED** [review]
Nice, I just found a few minor things that would be good to fix.

**Review by chrzaszcz — APPROVED** [review]
Looks good!

**Pull request merged**